app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,9 @@
+class CommentsController < ApplicationController
+  def create
+    post = Post.find(params[:id])
+    comment = current_user.comments.new(text: params[:text])
```

Comment:
- [ ] I would recommend using the `strong parameter` so that you can make your application more secure.

```ruby

class CommentsController < ApplicationController
  def create
    comment = current_user.comments.new(comment_params)
    ...
    ...
  end

  private
  
  def comment_params
    params.require(:comment).permit(:text)
  end
end
```